keywords/CouchbaseServer.py

```diff
@@ -250,3 +267,212 @@ def get_server_docs_with_prefix(self, url, bucket, prefix):
                 found_ids.append(row["$1"]["id"])
 
         return found_ids
+
+    def _wait_for_rebalance_complete(self, url):
+        """
+        Polls couchbase server tasks endpoint for any running rebalances.
+        Exits when no rebalances are in running state
```

Comment:
Does it wait until a rebalance is detected, and _then_ completes?  Or does it just see that there are no rebalances and then calls it a day?

I'm just wondering if there is the possibility for the following race:

- initiate rebalance
- call _wait_for_rebalance_complete
- rebalance starts